Remotion/Development/Web/UnitTesting/UI/Controls/FakeFallbackNavigationUrlProvider.cs

```diff
@@ -0,0 +1,29 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
```

Comment:
The two current commits can be flattened.

If separate commits should be done, they would be to separate the new classes and the integration.